crypto_kem/ntruhps4096821/clean/sample.c

```diff
@@ -1,27 +1,30 @@
-#include "crypto_sort.h"
 #include "sample.h"
 
 void PQCLEAN_NTRUHPS4096821_CLEAN_sample_fg(poly *f, poly *g, const unsigned char uniformbytes[NTRU_SAMPLE_FG_BYTES]) {
+
     PQCLEAN_NTRUHPS4096821_CLEAN_sample_iid(f, uniformbytes);
     PQCLEAN_NTRUHPS4096821_CLEAN_sample_fixed_type(g, uniformbytes + NTRU_SAMPLE_IID_BYTES);
 }
 
 void PQCLEAN_NTRUHPS4096821_CLEAN_sample_rm(poly *r, poly *m, const unsigned char uniformbytes[NTRU_SAMPLE_RM_BYTES]) {
+
     PQCLEAN_NTRUHPS4096821_CLEAN_sample_iid(r, uniformbytes);
     PQCLEAN_NTRUHPS4096821_CLEAN_sample_fixed_type(m, uniformbytes + NTRU_SAMPLE_IID_BYTES);
 }
 
+
+#include "crypto_sort_int32.h"
```

Comment:
Should this not be at the top of this file? (I think one of the linters is supposed to not like this...)